packages/now-go/index.ts

```diff
@@ -135,7 +147,7 @@ Learn more: https://zeit.co/docs/v2/deployments/official-builders/go-now-go/#ent
       isGoModExist = true;
       isGoModInRootDir = true;
       goModPath = fileDirname;
-    } else if (file.includes('go.mod')) {
+    } else if (file.includes('go.mod') && !file.includes('vendor')) {
```

Comment:
```suggestion
    } else if (file.endsWith('go.mod') && !file.endsWith('vendor')) {
```

I think `includes` could cause problems. Consider a file named `vendor.js` which would match. Or maybe a better solution is to use `basedir()` instead.